source/opt/scalar_replacement_pass.cpp

```diff
@@ -536,6 +542,7 @@ bool ScalarReplacementPass::CheckType(const Instruction* typeInst) const {
         return false;
       return true;
     case SpvOpTypeArray:
+      if (IsSpecConstant(typeInst->GetSingleWordInOperand(1u))) return false;
```

Comment:
I know it is not done consistently, but I think we should move towards always adding braces, and splitting these into two lines.  I'm constantly rewriting these when I want to add break points inside the if-block.